Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -821,8 +824,16 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
   bool hasTotalCounts(true);
   bool haveWeights = false;
   auto firstPulseT = DateAndTime::maximum();
+
+  if (!NexusDescriptor::findAndOpenParentGroup(*m_file, classType))
+    g_log.warning()
```

Comment:
This warning might solve my question above. In which case you could just add a comment that no event data would be loaded.